CustomerData/CartTagging.php

```diff
@@ -181,6 +187,19 @@ private function updateNostoId()
     }
 
     /**
+     * Return the checksum for for the customer tagging i.e hashed cookie identifier or HCID for
+     * short. This is used to sign the tagging so that if it is in fact cached, the cookie and
+     * tagging signature won't match and we'll be able to show a warning.
```

Comment:
Do we show and warning? 
I would say 'and we'll be able to discard the cached data'